docs/AdviceForOssBeginners.md

```diff
@@ -0,0 +1,26 @@
+# Advice for OSS Beginners
+
+//TODO: flesh out these points
+
+
+## Pay attention to instructions
+
+Take your time to learn and follow the workflow to the letter. 
+Do not skip any steps because you think that step is "not important". 
+
+## Stick to given versions
+
+## Follow others
+
+## Do your homework
+
+Use Google, read trouble shooting guides, search issue tracker (including closed issues)
```

Comment:
This one point taken from the TEAMMATES contributor's orientation guide sums up my feelings when new contributors ask help for the most trivial of tasks, and they really should know:

> Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resource _(Google, StackOverflow, troubleshooting guides, issue tracker, etc.)_ when you cannot. _You will not make a good impression to the dev team members if you ask questions which are already answered in the mentioned resources._

Sentences in italic are my own addition. I apologize if I sound too harsh, but the idea is there.